app/src/main/kotlin/com/hippo/ehviewer/ui/main/DownloadCard.kt

```diff
@@ -173,40 +184,76 @@ fun DownloadCard(
                         )
                     }
                 }
-                Row(
-                    verticalAlignment = Alignment.CenterVertically,
+
+                // Make spacing consistent with gallery page
+                // The height of the action button should be 2 * height of Text composable + 1dp of vertical padding (2dp total)
+                MeasureUnconstrainedViewHeight(
                     modifier = Modifier.constrainAs(actionsRef) {
                         end.linkTo(parent.end)
                         bottom.linkTo(parent.bottom)
                     },
-                ) {
-                    val running = downloadState == DownloadInfo.STATE_WAIT || downloadState == DownloadInfo.STATE_DOWNLOAD
-                    val icon = remember {
-                        movableContentOf<Boolean> {
-                            Icon(imageVector = if (it) Icons.Default.Pause else Icons.Default.PlayArrow, contentDescription = null)
-                        }
-                    }
-                    if (selectMode) {
-                        Box(modifier = Modifier.minimumInteractiveComponentSize()) {
-                            icon(running)
+                    viewToMeasure = {
+                        Text(text = "", style = MaterialTheme.typography.labelLarge)
+                    },
+                ) { measuredHeight ->
+                    Row(
+                        verticalAlignment = Alignment.CenterVertically,
+                        modifier = Modifier.height(measuredHeight * 2 + 2.dp).offset(x = 8.dp)
+                    ) {
+                        val running =
+                            downloadState == DownloadInfo.STATE_WAIT || downloadState == DownloadInfo.STATE_DOWNLOAD
+                        val icon = remember {
+                            movableContentOf<Boolean> {
```

Comment:
@FooIbar 
Why the `Icon` needs to be inside `movableContentOf`?

Reply:
To prevent recomposition, it's not required tho, unlike the thumb.